### PR TITLE
Sync startup script from https://github.com/ifel/pnh

### PR DESCRIPTION
**Pull request by ifel**
It's oriented for small machines like t2.micro - 1 CPU, 1 GB RAM. Can easily handle ~5000 connections.

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -1,23 +1,36 @@
...
-apt-get update && apt-get install -y python3-pip python3-venv git htop
+sudo systemctl start amazon-ssm-agent
+# Ubuntu 20 uses snap
+sudo snap start amazon-ssm-agent
+# MAX_TIME=20
+# MIN_TIME=13
+# DIFF=$(($MAX_TIME-$MIN_TIME+1))
+# R=$(($(($RANDOM%$DIFF))+$MIN_TIME))
+# sudo shutdown +$R
+sudo yum install -y python3-pip git htop
```
Comment (Yorkerrr) [inline script.sh]: we dont need tum here, we assume we run on ubunty

File: script.sh [hunk 2]
```diff
@@ -1,23 +1,36 @@
 #!/bin/bash
 set -x 
-apt-get update && apt-get install -y python3-pip python3-venv git htop
+sudo systemctl start amazon-ssm-agent
```
Comment (Yorkerrr) [inline script.sh:3]: please make this optional if 
`curl --max-time 3 http://169.254.169.254/latest/meta-data/ami-id 2>/dev/null 1>/dev/null`` return 0
same for next line

Comment (ifel) [reply script.sh:3]: Honestly, this does not matter. The script does not have set -e, so errors will just be ignored.

File: script.sh (hunk 1 above)
Comment (ifel) [reply script.sh]: Removed

**Pull request merged**